maintainers/maintainer-list.nix

```diff
@@ -1261,6 +1261,12 @@
     githubId = 56009;
     name = "Arcadio Rubio García";
   };
+  arcayr = {
```

Comment:
Put this commit before all others, because this is a dependency of the other one.